osu.Game/Overlays/SettingsPanel.cs

```diff
@@ -187,7 +180,7 @@ protected override void PopOut()
 
         protected override void OnFocus(FocusEvent e)
         {
-            searchTextBox.TakeFocus();
+            searchTextBox?.TakeFocus();
```

Comment:
I think this null-check is no longer required since adb4fd5? Just struck me as a bit of an odd change since no other accesses to `searchTextBox` are null-checked.